### PR TITLE
[NTVDM] Implement INT21, AX=4408, Determine if a block device is removable.

### PR DESCRIPTION
**Pull request by learn-more**
Found when trying Holmes2:
https://z.abandonware-france.org/jeux/1996/jeu-00796-sherlock_holmes_rose_tatouee-pcdos.7z

### CODE REVIEW

File: subsystems/mvdm/ntvdm/dos/dos32krnl/dosfiles.c [hunk 1]
```diff
@@ -1125,11 +1125,79 @@ BOOLEAN DosUnlockFile(WORD DosHandle, DWORD Offset, DWORD Size)
...
+            case DRIVE_REMOVABLE:
+            case DRIVE_CDROM:
+                *Result = 0x0000;
+                return TRUE;
+            case DRIVE_FIXED:
+                *Result = 0x0001;
+                return TRUE;
+            case DRIVE_REMOTE:
+            case DRIVE_RAMDISK: // ??
+                break;
```
Comment (HBelusca) [inline subsystems/mvdm/ntvdm/dos/dos32krnl/dosfiles.c]: Maybe these two should also fall back to the "default:" case?

File: subsystems/mvdm/ntvdm/dos/dos32krnl/dosfiles.c [hunk 2]
```diff
@@ -1125,11 +1125,79 @@ BOOLEAN DosUnlockFile(WORD DosHandle, DWORD Offset, DWORD Size)
...
+            }
+            Sda->LastErrorCode = ERROR_INVALID_FUNCTION;
+            return FALSE;
+        }
+        case 0x09:
+            DPRINT1("UNIMPLEMENTED INT 21h, 4409h, Determine if a logical device is local or remote %s\n", RootPath);
+            Sda->LastErrorCode = ERROR_INVALID_FUNCTION;
+            return FALSE;
+        default:
+            assert(0);
```
Comment (HBelusca) [inline subsystems/mvdm/ntvdm/dos/dos32krnl/dosfiles.c]: Not sure; just DPRINT that it's an unhandled case and fail as usual.

File: subsystems/mvdm/ntvdm/dos/dos32krnl/dosfiles.c (hunk 1 above)
Comment (learn-more) [reply subsystems/mvdm/ntvdm/dos/dos32krnl/dosfiles.c]: No clue, and I have no environment to test it.

File: subsystems/mvdm/ntvdm/dos/dos32krnl/dosfiles.c (hunk 2 above)
Comment (learn-more) [reply subsystems/mvdm/ntvdm/dos/dos32krnl/dosfiles.c]: No, ending up here is programmer error.

**Pull request merged**